utility/module_test.go

```diff
@@ -0,0 +1,123 @@
+package utility
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/pokt-network/pocket/persistence"
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/runtime/configs"
+	"github.com/pokt-network/pocket/runtime/test_artifacts"
+	"github.com/pokt-network/pocket/runtime/test_artifacts/keygen"
+	"github.com/pokt-network/pocket/shared/messaging"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/stretchr/testify/require"
```

Comment:
nit: import grouping:

```suggestion
	"log"
	"os"
	"testing"

	"github.com/stretchr/testify/require"

	"github.com/pokt-network/pocket/persistence"
	"github.com/pokt-network/pocket/runtime"
	"github.com/pokt-network/pocket/runtime/configs"
	"github.com/pokt-network/pocket/runtime/test_artifacts"
	"github.com/pokt-network/pocket/runtime/test_artifacts/keygen"
	"github.com/pokt-network/pocket/shared/messaging"
	"github.com/pokt-network/pocket/shared/modules"
```